File: dart/bin/shapes_app.dart
import "package:shapes/base/models.dart" show Circle;
import "package:shapes/client/shell.dart" show Runner;

void main() {
  Runner program = Runner();

  Circle circle = Circle(15.25);
  circle.color = "RED";
  circle.printDescription();

  program.run();
}

File: rust/src/bin/shapes_app.rs
use shapes::base::commons::BaseShape;
use shapes::base::models::Circle;
use shapes::client::shell::Runner;

fn main() {
    let program: Runner = Runner::new();

    let mut circle: Circle = Circle::new(15.25);
    circle.set_color("RED");
    circle.print_description();

    program.run();
}
